Luna/Luna/AddPeriodViewController.swift

```diff
@@ -104,6 +92,58 @@ class AddPeriodViewController: UIViewController
         setLabelDates()
     }
     
+    fileprivate func onSavePressed()
+    {
+        let canSave = checkDate()
+        
+        if canSave
+        {
+            createPeriodObject()
+            delegate?.onLastCycleChange()
+            delegate?.onDismissAddPeriod()
+        }
+    }
+    
+    fileprivate func checkDate() -> Bool
+    {
+        
+        let selectDate = addPeriodViewModel.Start
+        let endDate = addPeriodViewModel.End
+     
+        let order = Calendar.current.compare(selectDate, to: endDate, toGranularity: .day)
+        
+        switch order
+        {
+            case .orderedDescending:
+                let alert = UIAlertController( title: Constants.InterfaceBuilderStrings.pastDateTitle,
+                                               message: Constants.InterfaceBuilderStrings.pastDateMessage,
+                                               preferredStyle: .alert )
+                alert.addAction(UIAlertAction(title: "Ok", style: .default, handler: { (action: UIAlertAction!) in
```

Comment:
This handler should simply be `nil`. The default behavior for a button press with a UIAlertController is to dismiss itself. So, it's not really hurting anything, but this isn't needed.